clients/src/test/java/org/apache/kafka/clients/producer/KafkaProducerTest.java

```diff
@@ -1037,7 +1126,7 @@ public void testCloseIsForcedOnPendingAddOffsetRequest() throws InterruptedExcep
 
         Time time = new MockTime();
         MetadataResponse initialUpdateResponse = TestUtils.metadataUpdateWith(1, singletonMap("testTopic", 1));
-        ProducerMetadata metadata = newMetadata(0, Long.MAX_VALUE);
+        ProducerMetadata metadata = newMetadata(0, Long.MAX_VALUE, Long.MAX_VALUE);
```

Comment:
Is there a reason why every test seems to be using the same expiration and idle time? Earlier, we would have used default idle time?

Reply:
No reason. I've reverted the functionality to be as it was using 5 minutes for the test.